tee-worker/litentry/core/data-providers/src/lib.rs

```diff
@@ -108,24 +109,29 @@ impl DataProvidersStatic {
 		}
 	}
 	pub fn set_twitter_official_url(&mut self, v: String) {
+		debug!("set_twitter_official_url:{:?}", v);
 		self.twitter_official_url = v;
 	}
 	pub fn set_twitter_litentry_url(&mut self, v: String) {
+		debug!("set_twitter_litentry_url:{:?}", v);
 		self.twitter_litentry_url = v;
 	}
 	pub fn set_twitter_auth_token(&mut self, v: String) {
 		self.twitter_auth_token = v;
```

Comment:
why don't also add debug output for these three 'auth_token' set method?

Reply:
forgot it. :persevere: